src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.hpp

```diff
@@ -160,6 +160,47 @@ class jit_exp_emitter : public jit_emitter {
     void emit_isa(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const;
 };
 
+class jit_maximum_emitter : public jit_emitter {
```

Comment:
Seems we need test coverage about new added max and min emitters.

Reply:
`Minimum` and `Maximum` operation emitters are coverd by tests: `MaxMinLayerTest`, `ConversionLayerTest`, `FakeQuantizeLayerTest` and others. But used inference primitive was not checked. I added new test cases for `EltwiseLayerCPUTest`. Thanks for comment.